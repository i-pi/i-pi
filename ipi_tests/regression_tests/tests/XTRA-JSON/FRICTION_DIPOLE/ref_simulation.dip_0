 #EXTRAS(dipole)# Step:           0 
  0.00000000e+00   0.00000000e+00   0.00000000e+00
 #EXTRAS(dipole)# Step:           1 
 -3.50000000e-07  -0.00000000e+00  -0.00000000e+00
 #EXTRAS(dipole)# Step:           2 
 -2.80000000e-06  -1.00000000e-08  -0.00000000e+00
 #EXTRAS(dipole)# Step:           3 
 -9.43000000e-06  -2.00000000e-08  -0.00000000e+00
 #EXTRAS(dipole)# Step:           4 
 -2.22400000e-05  -5.00000000e-08  -1.00000000e-08
 #EXTRAS(dipole)# Step:           5 
 -4.31600000e-05  -1.00000000e-07  -1.00000000e-08
 #EXTRAS(dipole)# Step:           6 
 -7.40200000e-05  -1.80000000e-07  -2.00000000e-08
 #EXTRAS(dipole)# Step:           7 
 -1.16490000e-04  -2.90000000e-07  -3.00000000e-08
 #EXTRAS(dipole)# Step:           8 
 -1.72090000e-04  -4.30000000e-07  -5.00000000e-08
 #EXTRAS(dipole)# Step:           9 
 -2.42150000e-04  -6.10000000e-07  -6.00000000e-08
