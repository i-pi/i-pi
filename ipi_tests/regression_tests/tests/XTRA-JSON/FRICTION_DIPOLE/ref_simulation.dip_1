 #EXTRAS(dipole)# Step:           0 
  0.00000000e+00   0.00000000e+00   0.00000000e+00
 #EXTRAS(dipole)# Step:           1 
 -3.00000000e-08  -1.05000000e-06   1.00000000e-08
 #EXTRAS(dipole)# Step:           2 
 -2.70000000e-07  -8.34000000e-06   7.00000000e-08
 #EXTRAS(dipole)# Step:           3 
 -9.10000000e-07  -2.76900000e-05   2.30000000e-07
 #EXTRAS(dipole)# Step:           4 
 -2.17000000e-06  -6.41900000e-05   5.40000000e-07
 #EXTRAS(dipole)# Step:           5 
 -4.29000000e-06  -1.21790000e-04   1.01000000e-06
 #EXTRAS(dipole)# Step:           6 
 -7.52000000e-06  -2.03110000e-04   1.66000000e-06
 #EXTRAS(dipole)# Step:           7 
 -1.21600000e-05  -3.09210000e-04   2.50000000e-06
 #EXTRAS(dipole)# Step:           8 
 -1.85300000e-05  -4.39490000e-04   3.49000000e-06
 #EXTRAS(dipole)# Step:           9 
 -2.70000000e-05  -5.91740000e-04   4.60000000e-06
