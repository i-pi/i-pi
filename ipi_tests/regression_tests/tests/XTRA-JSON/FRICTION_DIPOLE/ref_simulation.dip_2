 #EXTRAS(dipole)# Step:           0 
  0.00000000e+00   0.00000000e+00   0.00000000e+00
 #EXTRAS(dipole)# Step:           1 
 -3.55000000e-06   4.00000000e-08  -5.10000000e-07
 #EXTRAS(dipole)# Step:           2 
 -2.83800000e-05   3.10000000e-07  -4.07000000e-06
 #EXTRAS(dipole)# Step:           3 
 -9.55900000e-05   1.04000000e-06  -1.35100000e-05
 #EXTRAS(dipole)# Step:           4 
 -2.25990000e-04   2.41000000e-06  -3.13300000e-05
 #EXTRAS(dipole)# Step:           5 
 -4.39910000e-04   4.57000000e-06  -5.94600000e-05
 #EXTRAS(dipole)# Step:           6 
 -7.57070000e-04   7.62000000e-06  -9.92000000e-05
 #EXTRAS(dipole)# Step:           7 
 -1.19642000e-03   1.15900000e-05  -1.51090000e-04
 #EXTRAS(dipole)# Step:           8 
 -1.77599000e-03   1.64600000e-05  -2.14880000e-04
 #EXTRAS(dipole)# Step:           9 
 -2.51282000e-03   2.21300000e-05  -2.89520000e-04
