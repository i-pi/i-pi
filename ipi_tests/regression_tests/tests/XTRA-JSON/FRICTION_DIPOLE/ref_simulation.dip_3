 #EXTRAS(dipole)# Step:           0 
  0.00000000e+00   0.00000000e+00   0.00000000e+00
 #EXTRAS(dipole)# Step:           1 
 -9.00000000e-08   6.10000000e-07  -0.00000000e+00
 #EXTRAS(dipole)# Step:           2 
 -6.90000000e-07   4.82000000e-06  -1.00000000e-08
 #EXTRAS(dipole)# Step:           3 
 -2.35000000e-06   1.60000000e-05  -5.00000000e-08
 #EXTRAS(dipole)# Step:           4 
 -5.60000000e-06   3.70600000e-05  -1.10000000e-07
 #EXTRAS(dipole)# Step:           5 
 -1.10100000e-05   7.02900000e-05  -2.10000000e-07
 #EXTRAS(dipole)# Step:           6 
 -1.91900000e-05   1.17160000e-04  -3.50000000e-07
 #EXTRAS(dipole)# Step:           7 
 -3.07700000e-05   1.78230000e-04  -5.50000000e-07
 #EXTRAS(dipole)# Step:           8 
 -4.64400000e-05   2.53120000e-04  -8.10000000e-07
 #EXTRAS(dipole)# Step:           9 
 -6.69500000e-05   3.40470000e-04  -1.14000000e-06
