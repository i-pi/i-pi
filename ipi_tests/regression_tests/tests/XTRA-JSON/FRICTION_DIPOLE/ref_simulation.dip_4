 #EXTRAS(dipole)# Step:           0 
  0.00000000e+00   0.00000000e+00   0.00000000e+00
 #EXTRAS(dipole)# Step:           1 
 -1.30000000e-07  -3.50000000e-07  -0.00000000e+00
 #EXTRAS(dipole)# Step:           2 
 -1.03000000e-06  -2.79000000e-06  -1.00000000e-08
 #EXTRAS(dipole)# Step:           3 
 -3.47000000e-06  -9.27000000e-06  -3.00000000e-08
 #EXTRAS(dipole)# Step:           4 
 -8.19000000e-06  -2.14600000e-05  -6.00000000e-08
 #EXTRAS(dipole)# Step:           5 
 -1.58900000e-05  -4.06200000e-05  -1.20000000e-07
 #EXTRAS(dipole)# Step:           6 
 -2.72500000e-05  -6.75700000e-05  -2.00000000e-07
 #EXTRAS(dipole)# Step:           7 
 -4.28800000e-05  -1.02540000e-04  -3.10000000e-07
 #EXTRAS(dipole)# Step:           8 
 -6.33400000e-05  -1.45220000e-04  -4.40000000e-07
 #EXTRAS(dipole)# Step:           9 
 -8.91200000e-05  -1.94700000e-04  -6.00000000e-07
