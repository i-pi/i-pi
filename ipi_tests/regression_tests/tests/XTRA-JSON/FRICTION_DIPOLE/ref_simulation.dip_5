 #EXTRAS(dipole)# Step:           0 
  0.00000000e+00   0.00000000e+00   0.00000000e+00
 #EXTRAS(dipole)# Step:           1 
  4.80000000e-07  -0.00000000e+00  -0.00000000e+00
 #EXTRAS(dipole)# Step:           2 
  3.85000000e-06  -1.00000000e-08  -0.00000000e+00
 #EXTRAS(dipole)# Step:           3 
  1.29500000e-05  -2.00000000e-08  -1.00000000e-08
 #EXTRAS(dipole)# Step:           4 
  3.04900000e-05  -4.00000000e-08  -1.00000000e-08
 #EXTRAS(dipole)# Step:           5 
  5.90400000e-05  -8.00000000e-08  -3.00000000e-08
 #EXTRAS(dipole)# Step:           6 
  1.00970000e-04  -1.40000000e-07  -5.00000000e-08
 #EXTRAS(dipole)# Step:           7 
  1.58380000e-04  -2.20000000e-07  -7.00000000e-08
 #EXTRAS(dipole)# Step:           8 
  2.33080000e-04  -3.20000000e-07  -1.00000000e-07
 #EXTRAS(dipole)# Step:           9 
  3.26550000e-04  -4.50000000e-07  -1.40000000e-07
