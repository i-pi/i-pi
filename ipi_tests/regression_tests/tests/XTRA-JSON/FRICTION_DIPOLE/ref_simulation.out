# column   1     --> step : The current simulation time step.
# column   2     --> conserved : The value of the conserved energy quantity per bead.
    0.00000000e+00     6.35780346e-03   
    1.00000000e+00     6.35782781e-03   
    2.00000000e+00     6.35790024e-03   
    3.00000000e+00     6.35801885e-03   
    4.00000000e+00     6.35818055e-03   
    5.00000000e+00     6.35838113e-03   
    6.00000000e+00     6.35861536e-03   
    7.00000000e+00     6.35887715e-03   
    8.00000000e+00     6.35915968e-03   
    9.00000000e+00     6.35945559e-03   
